client/examples/perf.rs

```diff
@@ -0,0 +1,466 @@
+use std::num::NonZeroU32;
+use std::path::PathBuf;
+use std::time::{Duration, Instant};
+
+use env_logger;
```

Comment:
redundant use (the same applies to `client.rs`

Reply:
Done